apps/home/src/globals.css

```
@@ -6,20 +6,27 @@
     font-family: var(--font-inter);
   }
 
   body {
-    background-color: #fbf7f0;
-    color: #1f1f1f;
+    background-color: var(--color-theme);
+    color: var(--color-primary);
+    font-feature-settings: "ss01", "ss02", "cv01", "zero";
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    text-rendering: optimizeLegibility;
```

Stylelint (17.9.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)